### PR TITLE
Conj 4900 service account has least privilege permissions

### PR DESCRIPTION
**Pull request by jtuttle**
https://ca-il-jira.il.cyber-ark.com:8443/browse/CONJ-4900

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+./policy/authn-k8s.yml
```
Comment (doodlesbykumbi) [inline .gitignore]: missing newline at EOF :p

Comment (doodlesbykumbi) [inline .gitignore]: missing newline at EOF :p

File: manifests/test-app-conjur-authenticator-role-binding.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+metadata:
+  name: test-app-conjur-authenticator-role-binding
+  namespace: {{ TEST_APP_PROJECT_NAME }}
+subjects:
+  - kind: ServiceAccount
+    name: default
+    namespace: {{ CONJUR_PROJECT_NAME }}
+roleRef:
+  kind: ClusterRole
+  name: pod-get-list-exec-test
```
Comment (doodlesbykumbi) [inline manifests/test-app-conjur-authenticator-role-binding.yaml]: this should be `conjur-authenticator` to reference the ClusterRole created in the deployment scripts

Comment (jtuttle) [reply manifests/test-app-conjur-authenticator-role-binding.yaml]: Ah right good catch. Thanks.

File: .gitignore (hunk 1 above)
Comment (jtuttle) [reply .gitignore]: Hah, fixed.

**Pull request merged**